azure-iot-device/azure/iot/device/iothub/models/properties.py

```diff
@@ -0,0 +1,43 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module contains classes related to Digital Twin properties and components.
+"""
+from typing import Union, Dict, Any
+
+
+class Component(object):
+    def __init__(self, properties: Dict[str, Any] = {}):
```

Comment:
Is this syntax compatible with Python 2? I know there's some weirdness with typehints, although I'm not clear the exact extent to which it will present problems, as 2.7 does somewhat support it.